scipy/stats/stats.py

```diff
@@ -4385,17 +4385,23 @@ def mannwhitneyu(x, y, use_continuity=True, alternative='two-sided'):
     use_continuity : bool, optional
             Whether a continuity correction (1/2.) should be taken into
             account. Default is True.
-    alternative : 'less', 'two-sided', or 'greater'
+    alternative : None, 'less', 'two-sided', or 'greater'
             Whether to get the p-value for the one-sided hypothesis ('less'
-            or 'greater'), or for the two-sided hypothesis ('two-sided', is
-            the default)
+            or 'greater'), or for the two-sided hypothesis ('two-sided').
+            Defaults to None, which results in a p-value half the size of
+            the 'two-sided' p-value and a different U statistic. The default
+            behavior is not the same as using 'less' or 'greater', is not
+            recommended, and will soon be deprecated.
```

Comment:
I'd recommend to avoid making definite statements about the future like this :-). Maybe better state that it only exists for backwards compatibility and is not recommended for use in any new code.